docs/content/docs/deployment/config.md

```diff
@@ -34,6 +34,8 @@ The configuration is parsed and evaluated when the Flink processes are started.
 
 The out of the box configuration will use your default Java installation. You can manually set the environment variable `JAVA_HOME` or the configuration key `env.java.home` in `conf/flink-conf.yaml` if you want to manually override the Java runtime to use.
 
+You can specifiy a different configuration directory location by defining the `FLINK_CONF_DIR` environment variable. For resource-providers which provide non-session deployments, you can specifiy per-job configurations this way. Make a copy of the `conf` directory from the Flink distribution and modify the settings on a per-job basis. Note that this is not supported in Docker or standalone Kubernetes deployments.
```

Comment:
What happens if I set FLINK_CONF_DIR when submitting a job to a session cluster?

```suggestion
You can specify a different configuration directory location by defining the `FLINK_CONF_DIR` environment variable. For resource providers which provide non-session deployments, you can specify per-job configurations this way. Make a copy of the `conf` directory from the Flink distribution and modify the settings on a per-job basis. Note that this is not supported in Docker or standalone Kubernetes deployments.
```

Reply:
thanks for the feedback. Looks like something is not correctly wired in my 🧠  😂